src/main/kotlin/org/jitsi/nlj/RtpSenderImpl.kt

```diff
@@ -145,6 +148,7 @@ class RtpSenderImpl(
                 packetInfo
             }
             node(rtcpSrUpdater)
+            node(pcapWriter)
```

Comment:
The `pcapWriter` node can't be on both the RTP and RTCP pipelines.  When this is run, health checks fail with
```JVB 2019-12-31 15:39:26.429 SEVERE: [40] Health.doRun#300: Health check failed in 1ms:
java.lang.Exception: Attempt to replace a Node's child. If this is intentional, use detachNext first.
```

I agree we want the PCAP file to capture both RTP and RTCP, but the node architecture means that we'll need some sort of indirection from node to pcap writer.

Reply:
I've pushed https://github.com/jitsi/jitsi-media-transform/pull/169/commits/cda1ea291728c73589d40d90a128cacbaea88156 as a starting point for fixing this. We could probably merge the ToggleablePcapWriter and PcapWriter back together, i.e. drop the `ObserverNode` inheritance from the PcapWriter and move all the new methods into PcapWriter /cc @bbaldino